### PR TITLE
Updated text

### PR DESCRIPTION
**Pull request by elletjies**
Please delete this line and the text below before submitting your contribution.

---
Thanks for contributing! 

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

---

### CODE REVIEW

**Comment by maneesha** [comment]
Thanks @elletjies 
Checking in with @sheraaronhurt about one of your suggestions.  You recommend changing "teaching skill" to "teaching skills."  [See the change here](https://carpentries.github.io/instructor-training/18-preparation.html#building-teaching-skill).  However, in our curriculum, we do use the singular _skill_ -- [see example here](https://carpentries.github.io/instructor-training/18-preparation.html#building-teaching-skill).

We should be consistent throughout so I will defer to @sheraaronhurt about this.

**Comment by sheraaronhurt** [comment]
hey @maneesha thanks for bringing this up. Let's keep it singular to align with other places in the curriculum

File: content/trainer-training/_index.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ Instructor Trainer Training is the process by which we onboard new Instructor Tr
 <table>
     <tr>
         <td>Has Feature</td>
-        <td><b>Evidence-Based Teaching Practices.</b>Learn evidence-based teaching practices to be more effective both in Carpentries workshops and your teaching more broadly. </td>
+        <td><b>Evidence-Based Teaching Practices.</b>Learn evidence-based teaching practices to be more effective both in Carpentries workshops and in your teaching more broadly. </td>
     </tr>
     <tr>
         <td>Has Feature</td>
-        <td><b>Teaching Opportunities.</b> Continue to develop your own teaching skill by teaching Instructor Training Events. </td>
+        <td><b>Teaching Opportunities.</b> Continue to develop your teaching skills by teaching Instructor Training Events. </td>
```
Comment (sheraaronhurt) [inline content/trainer-training/_index.md:32]: keep singular

**Review by sheraaronhurt — APPROVED** [review]

**Comment by sheraaronhurt** [comment]
Everything is good to go with the exception of keeping teaching skill singular

**Pull request merged**